DigikalaCrawler/spiders/comment_spider.py

```diff
@@ -64,8 +66,16 @@ def parse_comments(self, response):
             pos = [x.strip() for x in pos_raw]
             neg_raw = comment_selector.css('.c-comments__evaluation-negative li::text').extract()
             neg = [x.strip() for x in neg_raw]
-            txt = comment_selector.css('p::text').extract_first()
+            
+            ##########MODIFIED BY ARMAN############
```

Comment:
You should delete this comment. People can use `git blame` or other editors' tools like **annotation** to see your contribution. Furthermore, your contributions would be always presented in repo's history.